content/docs/user-guide/glossary/dvc-project.md

```diff
@@ -0,0 +1,28 @@
+---
+name: 'DVC Project'
+match:
+  [
+    'DVC project',
+    'DVC projects',
+    project,
+    projects,
+    'DVC repository',
+    'DVC repositories',
+    repository,
+    repositories,
+  ]
+tooltip: >-
+  Initialized by running `dvc init` in the **workspace** (typically a Git
+  repository). It will contain the [`.dvc/`
+  directory](/doc/user-guide/dvc-files-and-directories), as well as `dvc.yaml`
+  and `.dvc` files created with commands such as `dvc add` or `dvc run`.
+---
+
+Initialized by running `dvc init` in the **workspace** (typically a Git
+repository). It will contain the
+[`.dvc/` directory](/doc/user-guide/dvc-files-and-directories), as well as
+`dvc.yaml` and `.dvc` files created with commands such as `dvc add` or
+`dvc run`.
+
+This page extends for a _while_, elaborating on what a DVC Project is in a way
+where it doesn't make sense to put **all** this stuff in a tooltip.
```

Comment:
Where can we see this page in the review app (link pls)? Thanks

Reply:
This was example copy that Roger added to show up the new functionality, that was merged into my branch. DVC Project is a glossary item but not in our list for Basic Concepts so it's not in the nav and I'll revert it to the previous form.

Reply:
Yep. This is no longer relevant. Resolving, thanks